### PR TITLE
Reqtorio Expansion V 1.4

### PR DESCRIPTION
**Pull request by Naaanii**
## About The Pull Request
Adds missing railgun ammo, SR-127 flak ammo, deployable floodlights and deployable cameras to reqtorio.
Also adds both plastiques and Detpacks to reqtorio.
Reworks the drop pod recipe into a generic equipment recipe. (Recipe unchanged)
## Why It's Good For The Game
More stuff, more fun.
## Changelog
:cl:
add: Added HVAP and smart railgun rounds to reqtorio.
add: Added SR-127 flak ammo to reqtorio.
add: Added deployable floodlights to reqtorio.
add: Added both plastiques to reqtorio.
add: Added detpacks to reqtorio.
spellcheck: Added a missing period to drop pod factory description.
/:cl:

### CODE REVIEW

**Comment by Woker2004** [comment]
C4 reqtorio when? but jokes aside it is decently nice. though the deployable floodlights are by far the best one added here

**Comment by Naaanii** [comment]
> C4 reqtorio when? but jokes aside it is decently nice. though the deployable floodlights are by far the best one added here

I can add that to this actually.

**Comment by Woker2004** [comment]
> I can add that to this actually.

holy based

**Review by Kuro020 — APPROVED** [review]

**Comment by TellyLsc** [comment]
reqtorioing floodlights and using them as glorified flares until xenos complain that the map is too bright

**Comment by FosterGrime-Heymynameslouie** [comment]
would also adding sandbags to reqtorio be alright?
cool pr nonetheless speedmrhg

File: code/modules/factory/parts.dm [hunk 1]
```diff
@@ -305,6 +346,22 @@ GLOBAL_LIST_INIT(railgun_magazine, list(
 /obj/item/factory_part/railgun_magazine/Initialize(mapload)
 	. = ..()
 	recipe = GLOB.railgun_magazine
+/obj/item/factory_part/railgun_hvap_magazine
```
Comment (TiviPlus) [inline code/modules/factory/parts.dm:349]:
```suggestion
	recipe = GLOB.railgun_magazine

/obj/item/factory_part/railgun_hvap_magazine
```

File: code/modules/factory/parts.dm [hunk 2]
```diff
@@ -305,6 +346,22 @@ GLOBAL_LIST_INIT(railgun_magazine, list(
...
+	name = "railgun round"
+	desc = "An unfinished magnetically propelled steel rod."
+	result = /obj/item/ammo_magazine/railgun/hvap
+
+/obj/item/factory_part/railgun_hvap_magazine/Initialize(mapload)
+	. = ..()
+	recipe = GLOB.railgun_magazine
+/obj/item/factory_part/railgun_smart_magazine
```
Comment (TiviPlus) [inline code/modules/factory/parts.dm:357]:
```suggestion
	recipe = GLOB.railgun_magazine

/obj/item/factory_part/railgun_smart_magazine
```

**Review by TiviPlus — CHANGES_REQUESTED** [review]
.

**Comment by Naaanii** [comment]
> would also adding sandbags to reqtorio be alright? cool pr nonetheless speedmrhg

I don't think any form of cades or defences should be in reqtorio as neat as they would be to have.

File: code/modules/factory/unboxer.dm [hunk 3]
```diff
@@ -227,6 +245,17 @@
...
+	desc = "A box with round metal plates inside. Used to refill Unboxers."
+	refill_type = /obj/item/factory_part/railgun_magazine
+	refill_amount = 20
+
+/obj/item/factory_refill/railgun_smart_magazine_refill
+	name = "box of rounded metal plates"
+	desc = "A box with round metal plates inside. Used to refill Unboxers."
+	refill_type = /obj/item/factory_part/railgun_magazine
+	refill_amount = 20
 /obj/item/factory_refill/minigun_powerpack_refill
```
Comment (TiviPlus) [inline code/modules/factory/unboxer.dm:259]:
```suggestion
	refill_amount = 20

/obj/item/factory_refill/minigun_powerpack_refill
```

**Comment by Woker2004** [comment]
Let us begin C4 trolling

**Pull request merged**